test/helpers/default-functions.ts

```diff
@@ -31,12 +46,12 @@ export function defaultFunctions(): void {
 	 */
 	browser.addCommand(
 		'dbQuery',
-		async ( query: string, config?: DatabaseConfig ) => {
+		( query: string, config?: DatabaseConfig ): string => {
 			if ( !config ) {
 				config = {
-					user: process.env.DB_USER,
-					pass: process.env.DB_PASS,
-					database: process.env.DB_NAME
+					user: testEnv.vars.DB_USER,
```

Comment:
Note that in most all cases where `process.env.*` was used in the past you should now use `testEnv.vars` where the env vars related to the services being tested are now loaded (by default from `variables.env`, `test/test-services.env`, and `local.env` -- in that order). Things related to the WDIO test runner, such as timeouts and log levels, are set in `test/test-runner.env` and referenced directly from `process.env`.